app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -309,3 +309,15 @@ h5,
     z-index: 1;
   }
 }
+
+// Overrides for search tips link and info icon
+// Prevents capitalization of search tips link text when the masthead has an image background
+.image-masthead .searchtips-link {
+  text-transform: none !important;
+}
+
+// Prevents underline on hover in the span containing the icon which includes white space.
+// This class requires display inline block on the span. 
+.blacklight-icons:hover {
+  text-decoration: none; 
+}
```

Comment:
Is this doing anything? I don't see any underlined whitespace when I remove this?

Reply:
It was displaying an underline earlier but now no longer is (perhaps due to upstream changes in Blacklight?)  I've removed this line since it no longer appears necessary.